packages/wrangler/src/sites.tsx

```diff
@@ -114,9 +115,16 @@ export async function syncAssets(
     return { manifest: undefined, namespace: undefined };
   }
 
+  if (dryRun) {
+    console.log("(note: doing a dry run, not uploading or deleting anything.)");
+  }
+
   const title = `__${scriptName}-workers_sites_assets${
     preview ? "_preview" : ""
   }`;
+
+  // Unfortunately, we'll create the namespace even if dryRun is true.
+  // This is be an acceptable tradeoff for now, and we can revisit later.
```

Comment:
Umm, why is this acceptable? And why do we need to do this?

Reply:
How about, if we are doing a dry-run, we just list all the assets that would be synced?

Reply:
Might as well just skip the whole thing.